modules/nf-core/vcflib/vcffilter/main.nf

```diff
@@ -0,0 +1,38 @@
+process VCFLIB_VCFFILTER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::vcflib=1.0.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vcflib:1.0.3--ha025227_0':
+        'quay.io/biocontainers/vcflib:1.0.3--ha025227_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '1.0.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    if ( !(args.contains("-f") || args.contains("--info-filter") || args.contains("-g") || args.contains("--genotype-filter")) ) {
+        error "VCFLIB_VCFFILTER requires either the -f/--info-filter or -g/--genotype-filter arguments to be supplied using ext.args."
+    }
+    """
+    vcffilter \\
+        $args \\
+        $vcf \\
+        | gzip --no-name > ${prefix}.filtered.vcf.gz
```

Comment:
Is there no way this can be compressed using `bgzip`? Most tools that require compressed VCFs don't support the normal GZIP format

Reply:
Good to know. It seems to be included in the vcflib conda/docker/singularity environments, so its an  easy change.